config/config_mgmt.py

```diff
@@ -0,0 +1,807 @@
+#!/usr/bin/env python
+#
+# config_mgmt.py
```

Comment:
The purpose of the file is a bit confusing. It appears to be a standalone executable script, yet classes are also being imported into other scripts. I think it makes more sense to separate the shared code out into a non-executable library file, and move the executable script out of this directory and into the scripts/ directory. This directory should only hold files which define subcommands of the config command.

Reply:
I removed the test functions, this script is used more like a library now.